cmd/clusterctl/examples/openstack/generate-yaml.sh

```diff
@@ -234,6 +234,12 @@ fi
 echo "---" >> $PWD/$OUTPUT/provider-components.yaml
 kustomize build $PWD/provider-component/clouds-secrets >> $PWD/$OUTPUT/provider-components.yaml
 echo "---" >> $PWD/$OUTPUT/provider-components.yaml
+
+# latest kustomize don't allow include files out of build folder
+cp -r $CONFIG_DIR/../../../../../../../vendor/sigs.k8s.io/cluster-api/config $PWD/provider-component/cluster-api
+echo "---" >> $PWD/$OUTPUT/provider-components.yaml
 kustomize build $PWD/provider-component/cluster-api >> $PWD/$OUTPUT/provider-components.yaml
```

Comment:
Should we change these calls to `kubectl kustomize` so we Font need the kustomize binary?

Reply:
yes, I will update